woocommerce/woocommerce
# Cleanup product task experiment

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/team-ghidorah/issues/131.

- Remove previous product & card layout  code 9619d4ec5489c2347b149538e5044467d712c581
- Rename ./experimental-products -> ./products a77340c1cbee12cc6c61aee8a894cfa17d86c319







### How to test the changes in this Pull Request:



1. Go to onboarding wizard's Business Features step
2. Set "Currently selling elsewhere" to "Yes, on other platform" and save
3. Go to WooCommerce > products task
4. Observe the import screen is displayed
5. Go to onboarding wizard's Business Features step
6. Set "Currently selling elsewhere" to "No"
7. Go to WooCommerce > products task
8. Observe the add product stacked layout screen is displayed

import product

<img width="1352" alt="Screen Shot 2022-12-13 at 15 15 33" src="https://user-images.githubusercontent.com/4344253/207252416-887429e3-a600-4c1d-a7bb-a3bf8ed2393c.png">

add product stacked layout

<img width="1352" alt="Screen Shot 2022-12-13 at 15 14 14" src="https://user-images.githubusercontent.com/4344253/207252420-6e52d7ee-7135-4d29-988c-6730ceb6f484.png">





### Other information:




### FOR PR REVIEWER ONLY:
